js/main.js

```diff
@@ -0,0 +1,29 @@
+//Helper functions
+function $q (cssQuery) {
+  return document.querySelector(cssQuery);
+};
+function $el (element) {
+  return document.createElement(element);
+};
+
+//Cached Nodes
+var theField = $q("#the-field");
+var theH1 = $q("#the-real-h1");
+var body = document.body;
+var theButn = $q("#the-butn");
```

Comment:
I highly encourage to come up with better the names than the ones I chose hastily. Something more closely represents the elements purpose which is not always easy, but that's one of the great difficulties of programming. :sweat_smile: